pkg/values/values.go

```diff
@@ -17,9 +17,13 @@ limitations under the License.
 package values
 
 import (
+	"context"
 	"helm.sh/helm/v3/pkg/chartutil"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
+// TODO: Consider deprecating Mapper and overrides in favour of Translator.
```

Comment:
Please let me know what do you think about this. Since `Translator` is a strictly more powerful interface, perhaps we should get rid of `Mapper`?

Reply:
+1 from me (pending a conversation about the timing of applying overrides)